test/v1/models/abbreviatedReturn/AbbreviatedReturnModelSpec.scala

```diff
@@ -30,21 +31,29 @@ class AbbreviatedReturnModelSpec extends WordSpec with Matchers with BaseConstan
 
       "max values given" in {
 
-        val expectedValue = abbreviatedReturnUltimateParentJson
+        val expectedValue = withoutAppointedReportingCompany(abbreviatedReturnUltimateParentJson)
         val actualValue = Json.toJson(abbreviatedReturnUltimateParentModel)
 
         actualValue shouldBe expectedValue
       }
 
       "min values given" in {
 
-        val expectedValue = abbreviatedReturnJsonMin
+        val expectedValue = withoutAppointedReportingCompany(abbreviatedReturnJsonMin)
         val actualValue = Json.toJson(abbreviatedReturnModelMin)
 
         actualValue shouldBe expectedValue
       }
     }
 
+    "not pass the appointedReportingCompany boolean" when {
```

Comment:
not pass - Is this correct?

Reply:
@AdamCoops yeah - the idea is that we only use this attribute for validation in our API application, we don't need to send it downstream. So this test confirms that if we parse this object into json we don't want to write the boolean in there.